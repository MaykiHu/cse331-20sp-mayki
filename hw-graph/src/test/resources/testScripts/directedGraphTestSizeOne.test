#Create a graph
CreateGraph graph1

#Add a node to graph
AddNode graph1 n1

#Gets size of one-node graph
GetSize graph1