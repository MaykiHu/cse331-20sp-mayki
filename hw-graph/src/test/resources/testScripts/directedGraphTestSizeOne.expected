#Create a graph
created graph graph1

#Add a node to graph
added node n1 to graph1

#Gets size of one-node graph
size of graph1 is 1